Makefile

```diff
@@ -178,6 +182,20 @@ bundle-build:
 .PHONY: deploy_bundle
 deploy_bundle: undeploy_bundle
 	operator-sdk run bundle $(IMAGE_NAMESPACE)/$(OPERATOR_NAME)-bundle:$(IMAGE_VERSION)
+ifeq ($(DISABLE_SERVICE_TLS), true)
+	@echo "Disabling TLS for in-cluster communication between Services"
+	@current_ns=`$(CLUSTER_CLIENT) config view --minify -o 'jsonpath={.contexts[0].context.namespace}'` && \
+	if [ -z "$${current_ns}" ]; then \
+		echo "Failed to determine Namespace in current context" >&2; \
+		exit 1; \
+	fi; \
+	set -f -- `$(CLUSTER_CLIENT) get sub -l "operators.coreos.com/$(OPERATOR_NAME).$${current_ns}" -o name` && \
+	if [ "$${#}" -ne 1 ]; then \
+		echo -e "Expected 1 Subscription, found $${#}:\n$${@}" >&2; \
+		exit 1; \
+	fi; \
+	$(CLUSTER_CLIENT) -n $(DEPLOY_NAMESPACE) patch --type=merge -p '{"spec":{"config":{"env":[{"name":"DISABLE_SERVICE_TLS","value":"true"}]}}}' "$${1}"
```

Comment:
The `contexts[0].context.namespace` check earlier will determine the current context namespace, but the `DEPLOY_NAMESPACE` may be set to a different value. ex. I can simply not set the `DEPLOY_NAMESPACE` and allow it to default to `container-jfr-operator-system`, and then `DISABLE_SERVICE_TLS=true make deploy_bundle` will succeed in performing a bundle deployment in my current namespace, but will fail to modify the `Subscription` because this command looks in the wrong namespace.

Reply:
Thanks for noticing. I had meant to remove the `-n $(DEPLOY_NAMESPACE)`. I think it should work as expected now.